app/views/mentee_profiles/dashboard.html.slim

```diff
@@ -45,9 +45,13 @@
 .col-xs-2.col-md-3.col-lg-2
   .mentor-name
     span Mentor:
-    span Sarah Jones
+    span = @mentor.first_name
   = image_tag "profile/mentor_icon.png", alt: "Mentor profile picture", class: "profile-icon"
-  span sarah@lorem.com
+  span 
+  = @mentor.email
   div
     span.badge.online &nbsp;
     | Online
+    - if !@mentor.is_missing
+      = button_to "Report Missing Mentor", { :action => "missing_mentor" }, :data => {:confirm => "Are you Sure to report your Mentor as missing?"}, :method => :missing_mentor,  class: "btn btn-pink"
```

Comment:
added the if statement on button, also added a confirmation box. 
replaced the other fields.